sdcm/logcollector.py

```diff
@@ -27,9 +30,14 @@ class CollectingNode():  # pylint: disable=too-few-public-methods
     def __init__(self, name, ssh_login_info=None, instance=None, global_ip=None):
         self.name = name
         self.remoter = RemoteCmdRunner(**ssh_login_info) if ssh_login_info else None
+        self._ssh_login_info = ssh_login_info
         self._instance = instance
         self.external_address = global_ip
 
+    @property
+    def ssh_login_info(self):
+        return self._ssh_login_info
```

Comment:
why not just name it ssh_login_info ? with no underscore ?

Reply:
yep. you right. Just repeated same as on BaseNode